### DISCUSSION TITLE
【内容产出挑战】token-vendor

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#41] [Amazing Things!]
悬赏价格：`¥600`
时间要求：`in this week`

针对 Challenges 2 写一篇包括合约分析的文章。

> https://github.com/scaffold-eth/scaffold-eth-typescript-challenges

> https://github.com/scaffold-eth/scaffold-eth-typescript-challenges/tree/challenge-2-token-vendor

**Comment by kenspirit** [comment]
写了[一版](https://github.com/WeLightProject/Web3-dApp-Camp/pull/46)，看好不好使吧。

**Comment by leeduckgo** [comment]
Done!

**Comment by leeduckgo** [comment]
Paid.
